src/Agent/NewRelic/Agent/Core/Agent.cs

```diff
@@ -427,30 +429,49 @@ public void RecordLogMessage(string frameworkName, object logEvent, Func<object,
             if (_configurationService.Configuration.LogEventCollectorEnabled)
             {
                 var logMessage = getLogMessage(logEvent);
-                if (string.IsNullOrWhiteSpace(logMessage))
+
+                var timestamp = getTimestamp(logEvent).ToUnixTimeMilliseconds();
```

Comment:
Would it make sense to move the getting and conversion of the timestamp below the short-circuit to avoid unnecessary work (however trivial)?

Reply:
That is a great catch! I'll make the adjustment.